libnmstate/varlink/nmstate_varlink.py

```diff
@@ -0,0 +1,244 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+import os
+import libnmstate
+import logging
+import errno
+import io
+import json
+import libnmstate.error as libnmError
+
+
+try:
+    import varlink
+
+except ModuleNotFoundError:
+    raise libnmError.NmstateDependencyError("python3 varlink module not found")
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+log_stringio = io.StringIO()
```

Comment:
There might be a race problem when multiple `get_logs()` been invoked which is actually possible as
varlink service is async.

Each `NmstateVarlinkService.Show()` should have its logger created and cleaned before return.
The same also applied to other `NmstateVarlinkService` methods.